disk-encryption-tool-dracut

```diff
@@ -3,6 +3,22 @@
 exec < /dev/console >/dev/console 2>&1
 type getarg > /dev/null 2>&1 || . /lib/dracut-lib.sh
 
+# get systemd credential
+# https://systemd.io/CREDENTIALS/
+get_credential()
+{
+	local var="${1:?}"
+	local name="${2:?}"
+	[ -n "$CREDENTIALS_DIRECTORY" ] || return 1
```

Comment:
Was this always launched as a systemd service? I though that is was registered as a traditional dracut module.

Reply:
no idea. according to my tests it's launched via the service file.